api/v1beta1/temporalcluster_types.go

```diff
@@ -85,6 +85,10 @@ type ServiceSpec struct {
 	// 7243 for Frontend service
 	// +optional
 	HTTPPort *int `json:"httpPort"`
+	// PProfPort defines a custom pprof port for the service.
+	// The port is defined by the global config.
+	// +optional
+	PProfPort *int `json:"pprofPort,omitempty"`
```

Comment:
IMO this field is useless. If  `spec.pprof` != nil the container will expose `spec.pprof.port`.